prometheus/src/main/scala/filodb/prometheus/ast/Vectors.scala

```diff
@@ -130,31 +144,27 @@ trait Vectors extends Scalars with TimeUnits with Base {
     * It is possible to filter these time series further by
     * appending a set of labels to match in curly braces ({}).
     */
-
-  case class InstantExpression(override val metricName: Option[String],
-                               override val labelSelection: Seq[LabelMatch],
+  case class InstantExpression(val metricName: Option[String],
```

Comment:
I actually like having the `override` marker because it helps reader know that this member implements a super-class/trait def without additional UI help.

Reply:
I find it confusing.  I'd rather have override there only if it is truly overriding something.  I suppose am open on this though